### ISSUE TITLE
Print Widget don't print only filtered rows

### ISSUE THREAD

**Issue by KayaCookie** [Mottie/tablesorter#1046]
Hi,

I've set up Tablesorter with 4 widgets : zebra,filter,formatter and print
All the widgets works fine, except for print widget that is printing all rows even if i declare print_rows:'f' in widgetOptions: {} ...

I don't understand why.

The print widget always print all the rows of my table...

Thanks in advance !

**Comment by KayaCookie** [comment]
![print_widget_bug](https://cloud.githubusercontent.com/assets/15120714/10478114/4af8f650-726c-11e5-9bd5-bee2d634268b.JPG)

**Comment by Mottie** [comment]
Hi @KayaCookie!

I honestly can't think of any reason why the `print_rows` option wouldn't work as expected.

Would you please share your tablesorter initialization code. Also, are you including any external stylesheet ([`print_styleSheet` option](http://mottie.github.io/tablesorter/docs/example-widget-print.html#print_stylesheet))? Including that might help narrow down the problem.

**Comment by KayaCookie** [comment]
Hi thanks for the answer. Here is my code : 

$(document).ready(function() 
			    { 

			    		var fullDate = new Date();
						var twoDigitMonth = fullDate.getMonth()+1+"";if(twoDigitMonth.length==1)	twoDigitMonth="0" +twoDigitMonth;
						var twoDigitDate = fullDate.getDate()+"";if(twoDigitDate.length==1)	twoDigitDate="0" +twoDigitDate;
						var currentDate = twoDigitDate + "-" + twoDigitMonth + "-" + fullDate.getFullYear();
						console.log(currentDate);

			        $("#tableau_mouvements").tablesorter(
			        	{
			        		sortList: [[4,0]],
			        		theme: "default",
			        		widgets: ["zebra","filter","formatter","print"],
			        		widthFixed: true,
			        		
			        		headers:
			        		{
			        			0: { filter: false},
								4: { sorter: "shortDate", dateFormat: "ddmmyyyy" },
								5: {filter: false},
								6: {filter: false},
								11: {filter: false}
							},

							widgetOptions : {
			        			
			        			filter_functions: {
			        				4: {
			        					"echu" : function(e, n, f, i, $r, c, data) { return n < $.now(); },
			        					"ce jour" : function(e, n, f, i, $r, c, data) { return e == currentDate; },
			        					"a venir" : function(e, n, f, i, $r, c, data) { return n > $.now(); }
			        				},
			        				7: {
			        					"Depassement" : function(e, n, f, i, $r, c, data) { return e > "00h00"; }
			        				},
			        			 	9: {
			        			 		"En cours" : function(e, n, f, i, $r, c, data) { return n != "ended"; },
			        					"Fini" : function(e, n, f, i, $r, c, data) { return n == "ended"; },
			        					"Active" : function(e, n, f, i, $r, c, data) { return n == "started"; },
			        					"En pause" : function(e, n, f, i, $r, c, data) { return n == "paused"; }
			        			 	}
			        			},
			        			formatter_column: {
			        				9: function( text, data ) {
							          if ( text === '' ) { return ''; }
							          // add text to data-attribute; this overrides the parser
							          data.$cell.attr( data.config.textAttribute, text );
							          // replace table cell with a link
							          return '<img src="/images/' + text + '.png"/>';
							        }
			        			},
			        			print_rows: 'f',
			        			print_columns: 'a'
			        		}
						}
			        	);

			        	 $('.print_table').click(function(){
							$('#tableau_mouvements').trigger('printTable');
						}); 
			    } 
			);

**Comment by KayaCookie** [comment]
And no i don't have any print-stylesheet options...

**Comment by Mottie** [comment]
Ok, I figured out the issue LOL.

In the demos I provided, I always include the tablesorter theme as a print style:

```js
print_styleSheet : '../css/theme.blue.css'
```

This adds the css needed to hide the filter row in the print preview.

I'm making a change to the print widget, in the master branch, which now includes the css to hide the filter row without needing the theme.

Sorry about taking so long, I have been very busy and distracted on the weekdays.
